src/chttpd/src/chttpd_db.erl

```diff
@@ -1816,7 +1816,14 @@ parse_changes_query(Req) ->
         {"heartbeat", "true"} ->
             Args#changes_args{heartbeat=true};
         {"heartbeat", _} ->
-            Args#changes_args{heartbeat=list_to_integer(Value)};
+            try list_to_integer(Value) of
+                HeartbeatInteger when HeartbeatInteger > 0 ->
+                    Args#changes_args{heartbeat=HeartbeatInteger};
+                _ ->
+                    throw({bad_request, <<"Negative heartbeat value">>})
```

Comment:
The "Negative heartbeat value" doesn't sound correct if user pass `0`. Should we change the wording to something like:
- The heartbeat value should be positive integer